recipes/default.rb

```diff
@@ -49,6 +49,7 @@
   'uri-stripper'            => node['repose']['uri_stripper']['cluster_id'],
   'add-header'              => node['repose']['add_header']['cluster_id']
```

Comment:
Sadly, it looks like someone merged some broken recipes prior to this PR.  Theres no trailing comma here.

![](https://m.popkey.co/daeb01/L9oQv_s-200x150.gif)

Or on the next line. I would true that up here.